environment.yaml

```diff
@@ -27,6 +27,10 @@ dependencies:
   - progressbar2
   - tqdm
 
+  # dev tools
+  - black
+  - pre-commit
```

Comment:
See notes to `setup.py`:
- `black` not required
- `pre-commit` here (`environment.yaml` = development environment)?